manifests/x/XmacsLabs/Mogan/1.2.0/XmacsLabs.Mogan.locale.zh-CN.yaml

```diff
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.2.10 $debug=QUSU.CRLF.7-4-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.5.0.schema.json
+
+PackageIdentifier: XmacsLabs.Mogan
+PackageVersion: 1.2.0
+PackageLocale: zh-CN
+Publisher: Xmacslabs
+PublisherUrl: https://gitee.com/XmacsLabs
+PublisherSupportUrl: https://gitee.com/XmacsLabs/mogan/issues
+# PrivacyUrl:
+Author: 沈浪熊猫儿
+PackageName: MoganResearch
+PackageUrl: https://mogan.app/
+License: GPL-3.0
+LicenseUrl: https://gitee.com/XmacsLabs/mogan/blob/branch-1.2/LICENSE
+Copyright: © 2020-2023 by Xmacs Labs
+CopyrightUrl: https://gitee.com/XmacsLabs/mogan/blob/branch-1.2/COPYING
+ShortDescription: 基于 GNU TeXmacs 的结构化编辑器
+Description: |-
+  墨干编辑器是墨者实验室旗下的结构化编辑器。
+  墨干编辑器基于 GNU TeXmacs 研发定制的自由软件。
+Agreements:
+  - AgreementUrl: https://github.com/XmacsLabs/mogan/blob/branch-1.2/LICENSE
```

Comment:
```suggestion
  - AgreementLabel: GNU GENERAL PUBLIC LICENSE
    AgreementUrl: https://github.com/XmacsLabs/mogan/blob/branch-1.2/LICENSE
```

Reply:
It's not necessary, see https://github.com/microsoft/winget-pkgs/pull/139478. But it's better to add one. Though I think `License agreement` is more suitable of a label here? (since it conveys what type of an agreement it is)

Reply:
Got the translation for `GNU GENERAL PUBLIC LICENSE` from https://www.gnu.org/licenses/old-licenses/gpl-2.0.zh-cn.html